annotation-file-utilities/tests/AnnotationEnumFieldInsertion.goal

```diff
@@ -0,0 +1,6 @@
+public class AnnotationEnumFieldInsertion {
+    void test() {
+        @fakeChecker.qual.Annotation(values={fakeChecker.qual.EnumClass.ENUM_CONSTANT})
+        int i = 1;
+    }
+}
```

Comment:
Please add a newline add the end of files. Check for the red mark shown in github.

Reply:
Done.